src/locales/en.json

```diff
@@ -361,7 +361,7 @@
       "instruction": "Remember to press stop when you're done. <fallback>Redo video actions</fallback>"
     },
     "camera_inactive": {
-      "message": "Camera not working?",
+      "message": "Having camera problems?",
```

Comment:
We need to add this to the Migration file

Reply:
I'm holding off updating Migration file for now pending confirmation from @raemarielee for which is the right title.

Reply:
```suggestion
      "message": "Camera not working",
```